common/keys.go

```diff
@@ -228,4 +228,7 @@ const (
 
 	// ArgoCDDexSecretKey is used to reference Dex secret from Argo CD secret into Argo CD configmap
 	ArgoCDDexSecretKey = "oidc.dex.clientSecret"
+
+	// Label Selector is an env variable for ArgoCD instance reconcilliation.
+	ArgoCDLabelSelector = "ARGOCD_LABEL_SELECTOR"
```

Comment:
`ArgoCDLabelSelector` can be renamed a `ArgoCDLabelSelectorKey`, like the previous ones and comments should start with the name of the variable.